Week8/app/src/main/java/com/isabelledionisius/myomelettapp/omeletts/model/OmelettViewModel.kt

```diff
@@ -0,0 +1,44 @@
+package com.isabelledionisius.myomelettapp.omeletts.model
+
+import android.app.Application
+import androidx.lifecycle.AndroidViewModel
+import androidx.lifecycle.LiveData
+import com.isabelledionisius.myomelettapp.repository.OmelettRepository
+import com.isabelledionisius.myomelettapp.db.OmelettDataList
+import com.isabelledionisius.myomelettapp.db.OmelettRoomDatabase
+import com.isabelledionisius.myomelettapp.omeletts.model.response.OmelettData
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.coroutineScope
+import kotlinx.coroutines.launch
+
+class OmelettViewModel(application: Application) : AndroidViewModel(application) {
+
+    private val repository: OmelettRepository
```

Comment:
And then I'm not sure why you have the OmelettViewModel, and a repository, and you're still building the repo in here (one of the previous comments), and also you're using the Repo here and the ViewModel, but not for the RemoteApi.

Please make sure your code is more consistent in the future!